clipper_admin/clipper_manager.py

```diff
@@ -160,37 +160,35 @@ def __init__(self,
         }
         start_redis = (self.redis_ip == DEFAULT_REDIS_IP)
         if start_redis:
-            self.docker_compost_dict['services']['redis'] = {
-                'image': 'redis:alpine',
+            self.docker_compose_dict['services']['redis'] = {
+                'image':
+                'redis:alpine',
                 'ports': ['%d:%d' % (self.redis_port, self.redis_port)],
-                'command': "redis-server --port %d" % self.redis_port,
+                'command':
+                "redis-server --port %d --appendonly yes" % self.redis_port,
+                # Set the user id and group id to set command execution privileges
+                'user':
+                "%s:%s" % (str(os.getuid()), str(os.getgid())),
```

Comment:
I believe the reason this is failing is that the user running this command is often not the same user and doesn't have the same permissions as the docker user. For example, on Jenkins `os.getuid()` would return the uid for the `jenkins` user, but the Docker daemon runs as `root`.